config/application.rb

```diff
@@ -144,7 +144,11 @@ class Application < Rails::Application
     )
 
     # Check that the loggers setting exist to configure the app and sql loggers
-    Foreman::Logging.add_loggers(SETTINGS[:loggers] || {:app => {:enabled => true}, :sql => {:enabled => true}})
+    Foreman::Logging.add_loggers(SETTINGS[:loggers] || {
+      :app => {:enabled => true},
+      :sql => {:enabled => true},
+      :crs => {:enabled => true}
```

Comment:
This reminded me to move the LDAP stuff to a separate logger too, in #2437, which I switched off by default and improved enabling a single logger from settings.yaml.